docs/user/user-defined-redirects.rst

```diff
@@ -73,23 +78,18 @@ You would set the following configuration::
     From URL: /example.html
     To URL: /examples/intro.html
 
-Note that the ``/`` at the start doesn't count the ``/$lang/$version`` prefix (e.g.
-``/en/latest``), but just the user-controlled section of the URL.
+**Page Redirects apply to all versions of you documentation.**
+Because of this,
+the ``/`` at the start of the ``From URL`` doesn't include the ``/$lang/$version`` prefix (e.g.
+``/en/latest``), but just the version-specific part of the URL.
 If you want to set directs only for some languages or some versions, you should use
 :ref:`user-defined-redirects:exact redirects` with the fully-specified path.
 
-.. tip::
-
-   *Page Redirects* can redirect URLs **outside** Read the Docs platform
-   just by defining the "To URL" as the absolute URL you want to redirect to.
```

Comment:
Are you removing these on purpose?

Reply:
Yes, they were moved to the top, instead of being repeated twice.